docs/zk2kraft.html

```diff
@@ -15,14 +15,13 @@
  limitations under the License.
 -->
 
-<script><!--#include virtual="js/templateData.js" --></script>
+<!--#include virtual="../includes/_header.htm" -->
+<!--#include virtual="../includes/_top.htm" -->
 
-<script id="zk2kraft-template" type="text/x-handlebars-template"></script>
 
+<div class="content">
 <div class="p-zk2kraft">
-    <h4 class="anchor-heading">Significant Changes in Kafka 4.0 Release</h4>
-    <p>The following are some of the updates in Kafka 4.0 release:</p>
-    <h5 class="anchor-heading">Removal Zookeeper configs</h5>
+    <h2 class="anchor-heading">Removal Zookeeper Configurations</h2>
```

Comment:
If a hyperlink is desired in the comment above https://github.com/apache/kafka/pull/18961/files#r1960998479, we could add it here.

```suggestion
    <h2 class="anchor-heading"><a id="upgrade-significant-changes" class="anchor-link"></a><a href="#upgrade-significant-changes">Removal Zookeeper Configurations</a></h2>
```